src/server/ocpp/services/OCPPService.ts

```diff
@@ -633,8 +642,12 @@ export default class OCPPService {
       await OCPPUtils.priceTransaction(headers.tenantID, transaction, consumption, TransactionAction.START);
       // Billing
       await OCPPUtils.billTransaction(headers.tenantID, transaction, TransactionAction.START);
+      // Is it possible that a TagID is registered at Gireve and Hubject? Then only one Session for either Gireve or Hubject should be started.
+      // Assumption: Either Gireve or Hubject is enabled for eRoaming
       // OCPI
       await OCPPUtils.processOCPITransaction(headers.tenantID, transaction, chargingStation, TransactionAction.START);
```

Comment:
You have here to do the same as before. 
I think you should define an abstract class for roaming hub types and only call the abstract methods here.

Reply:
For now I just added the if condition, but I will work on the abstract class in the future